components/concept/sync/src/main/java/mozilla/components/concept/sync/Devices.kt

```diff
@@ -68,9 +68,10 @@ interface DeviceConstellation : Observable<AccountEventsObserver> {
      * Send a command to a specified device.
      * @param targetDeviceId A device ID of the recipient.
      * @param outgoingCommand An event to send.
-     * @return A [Deferred] that will be resolved with a success flag once operation is complete.
+     * @return A [Deferred] that will be resolved when the operation is complete.
+     * @throws Exception if the operation could not be completed.
```

Comment:
It might be worth clarifying that it will (should?) only throw `FxaException` exceptions (here and below)

Reply:
Thanks!
Changed `Exception` occurrences to `FxaException`.